src/components/unsigned-sweep.js

```diff
@@ -217,13 +218,26 @@ class UnsignedSweep extends Component {
   }
 
   render() {
-    const recoveryCoins = coinConfig.supportedRecoveries.unsignedSweep;
+    const recoveryCoins = coinConfig.supportedRecoveries.unsignedSweep[this.state.env];
     const { isLoggedIn } = this.props;
-
+    let warning;
+    if (this.state.coin === 'bsv') {
+      warning =
+      <Alert color='danger'>
+        <p>
+          Bitcoin SV Transactions are replayable on the Bitcoin Cash Network.
+        </p>
+        <p>
+          Please make sure you are the owner of the Destination Address to avoid
+          accidentally sending your BCH to an address you don't own.
+        </p>
+      </Alert>
+    }
```

Comment:
this looks good but note that at least in FE development, HTML typically uses double quotes for inline html

Reply:
this is more of a FE style thing than anything